corehq/sql_db/routers.py

```diff
@@ -55,7 +56,8 @@ def allow_migrate(db, app_label):
     """
     if app_label == ICDS_REPORTS_APP:
         db_alias = get_icds_ucr_db_alias()
-        return bool(db_alias and db_alias == db)
+        non_dashboard_alias = get_icds_ucr_non_dashboard_db_alias()
+        return bool(db_alias and db_alias == db) or bool(non_dashboard_alias and non_dashboard_alias == db)
```

Comment:
Is this necessary? I think that all the UCR stuff will be handled by SQLAlchemy and this is just for django routing right?

Reply:
This is to determine if any migrations defined in ICDS_REPORTS_APP should be run on this new DB too or not. I assumed there would be some django migrations that might need to be run, but if that's not true, I can remove it.

Reply:
Removed this as well, per the [comment](https://github.com/dimagi/commcare-cloud/pull/2623#discussion_r255578821)